sdk/client-core/core/src/main/java/io/clientcore/core/util/ClientLogger.java

```diff
@@ -549,88 +489,11 @@ private String getMessageWithContext(String message) {
         }
 
         private void addKeyValueInternal(String key, Object value) {
-            if (this.context == null) {
-                this.context = new ArrayList<>();
-            }
-
-            this.context.add(new ContextKeyValuePair(key, value));
-        }
-
-        private void performLogging(LogLevel logLevel, String message, Throwable throwable) {
-            switch (logLevel) {
-                case VERBOSE:
-                    logger.debug(message, throwable);
-                    break;
-                case INFORMATIONAL:
-                    logger.info(message, throwable);
-                    break;
-                case WARNING:
-                    logger.warn(message, throwable);
-                    break;
-                case ERROR:
-                    logger.error(message, throwable);
-                    break;
-                default:
-                    // Don't do anything, this state shouldn't be possible.
-                    break;
-            }
-        }
-
-        /**
-         * Serializes passed map to string containing valid JSON fragment:
-         * e.g. "k1":"v1","k2":"v2", properly escaped and without trailing comma.
-         * <p>
-         * For complex object serialization, it calls {@code toString()} guarded with null check.
-         *
-         * @param context to serialize.
-         *
-         * @return Serialized JSON fragment or an empty string.
-         */
-        static String writeJsonFragment(Map<String, Object> context) {
```

Comment:
Same question on why some of this code has been removed. It's higher level than SLF4J

Reply:
we, following your suggestion, now rely more on json serialization and not our hacky json string writing